src/tracks/playabletrack/wavetrack/ui/SpectrumView.cpp

```diff
@@ -921,3 +922,34 @@ PopupMenuTable::AttachedItem sAttachment{
 };
 }
 
+static bool ShouldCaptureEvent(wxKeyEvent& event, SpectralData *pData)
+{
+   const auto keyCode = event.GetKeyCode();
+   return
+      (keyCode == WXK_BACK || keyCode == WXK_DELETE ||
+       keyCode == WXK_NUMPAD_DELETE)
+      && pData && !pData->dataHistory.empty();
```

Comment:
Affordance control of wave track also handles entrer and back keys when in name edit mode, seems like these events not handled any more by affordance